Quintet-iOS/View/ElementView.swift

```diff
@@ -0,0 +1,164 @@
+                        .font(.system(size: 15))
+                }
+                .padding(.top, 25)
+                .padding(.trailing, 180)
+            }
+            .sheet(isPresented: $isShowPopup) {
+                CalendarYearMonthPickerPopup(viewModel: viewModel, isShowPopup: $isShowPopup)
+                    .frame(width: 300, height: 400)
+                    .background(BackgroundClearView())
+                    .ignoresSafeArea()
+            }
+            ForEach(records) { metaData in
+                ForEach(metaData.records) { record in
+                    recordCard(icon: record.icon, title: record.title, subtitle: record.subtitle)
+                }
+                .padding(.horizontal, 20)
+            }
+        }
+    }
+    
+    var body: some View {
+        VStack(spacing: 6) {
+            HStack(spacing: 6) {
+                Button(action: {
+                    self.recordIndex = .work
+                }) {
+                    ElementCard(icon: "pencil", title: "일", size: 34, space: 55, fC: recordIndex == .work ? Color.white : Color("DarkGray"), bC: recordIndex == .work ? Color("DarkQ") : Color.white)
+                }
+                
+                Button(action: {
+                    self.recordIndex = .health
+                }) {
+                    ElementCard(icon: "cross.circle.fill", title: "건강", size: 30, space: 30, fC: recordIndex == .health ? Color.white : Color("DarkGray"), bC: recordIndex == .health ? Color("DarkQ") : Color.white)
+                }
+            }
+            
+            HStack(spacing: 6) {
+                Button(action: {
+                    self.recordIndex = .relation
+                }) {
+                    ElementCard(icon: "person.3.fill", title: "관계", size: 25, space: 25, fC: recordIndex == .relation ? Color.white : Color("DarkGray"), bC: recordIndex == .relation ? Color("DarkQ") : Color.white)
+                }
+                
+                Button(action: {
+                    self.recordIndex = .family
+                }) {
+                    ElementCard(icon: "heart.fill", title: "가족", size: 30, space: 30, fC: recordIndex == .family ? Color.white : Color("DarkGray"), bC: recordIndex == .family ? Color("DarkQ") : Color.white)
+                }
+            }
+            
+            HStack(spacing: 230) {
+                Button(action: {
+                    self.recordIndex = .money
+                }) {
+                    Text("자산")
+                        .fontWeight(.bold)
+                        .font(.system(size: 23))
+                        .foregroundColor(recordIndex == .money ? Color.white : Color("DarkGray"))
+                        .offset(x: -113)
+                    
+                    Image(systemName: "dollarsign.circle.fill")
+                        .font(.system(size: 30))
+                        .foregroundColor(recordIndex == .money ? Color.white : Color("DarkGray"))
```

Comment:
```suggestion
                        .foregroundColor(recordIndex == .money ? .white : Color("DarkGray"))
```
마찬가지로 Color는 생략해도 괜찮을 것 같습니다!